cookbooks/aws-parallelcluster-computefleet/kitchen.computefleet-install.yml

```diff
@@ -15,7 +15,7 @@ suites:
   - name: clusterstatusmgtd
     run_list:
       - recipe[aws-parallelcluster-tests::setup]
-      - recipe[aws-parallelcluster-computefleet::clusterstatusmgtd]
+      - recipe[aws-parallelcluster-computefleet::clusterstatusmgtd_install]
```

Comment:
Thought: is it worth transforming it into a resource with 2 actions?

Reply:
Yes, we can think to do it in future.